src/plugins/visualizations/common/expression_functions/vis_dimension.ts

```diff
@@ -25,7 +25,7 @@ export type ExpressionValueVisDimension = ExpressionValueBoxed<
   'vis_dimension',
   {
     accessor: number | DatatableColumn;
-    format: {
+    format?: {
```

Comment:
Why this change?

Reply:
Good question, it started when I saw this function...

https://github.com/elastic/kibana/blob/f062bf8638f251463bcdc3aec4e5a336123bcedd/src/plugins/visualizations/common/utils/accessors.ts#L100-L108

Namely the `dimension.format || defaultColumnFormat` part that if pointless as `format` is always defined. So I tested out making it optional.

It's strange to me to have an object with all optional values and not have the value itself as optional. I just reverted the change to avoid any unforeseen issues, see f062bf8638f251463bcdc3aec4e5a336123bcedd.